lib/Mojo/WebSocketProxy/CallingEngine.pm

```diff
@@ -144,7 +144,12 @@ sub call_rpc {
 
             my $api_response;
             if (!$res) {
-                warn "WrongResponse [$msg_type]";
+                my $tx = $client->tx;
+                my $details = 'URL: ' . $tx->req->url;
```

Comment:
Thanks for paying attention to that!

According to the implementation of `MojoX::JSON::RPC::Client` we shouldn't: 

- `$callback` is invoked at  https://metacpan.org/source/HENRYYKT/MojoX-JSON-RPC-0.10/lib/MojoX/JSON/RPC/Client.pm#L47 after doing `_process_result`

- in `_process_result` it already assumes `$tx` is defined https://metacpan.org/source/HENRYYKT/MojoX-JSON-RPC-0.10/lib/MojoX/JSON/RPC/Client.pm#L104 . I.e. if it will be undefined, it will `die` in non-our code already.